dbcon/mysql/ha_mcs_impl.cpp

```diff
@@ -1821,7 +1854,7 @@ uint32_t doUpdateDelete(THD* thd, gp_walk_info& gwi, const std::vector<COND*>& c
 
             try
             {
-                colrids = csc->columnRIDs(deleteTableName);
+                colrids = csc->columnRIDs(deleteTableName, lower_case_table_names);
```

Comment:
We are missing the 2nd argument here, which should be false.

Reply:
Fixed